odpi.bom

```diff
@@ -132,7 +132,16 @@ bigtop {
                 ref = 'branch-2.7-ODP'; dir = 'branch-2.7-ODP'}
 ***/
     }
-
+    'hive' {
+      name    = 'hive'
+      relNotes = 'Apache Hive'
+      version { base = '1.2.1'; pkg = base+'+odpi'; release = 1 }
+      tarball { destination = "apache-${name}-${version.base}-src.tar.gz"
+                source      = destination }
+      url     { download_path = "/$name/$name-${version.base}/"
+                site = "${apache.APACHE_MIRROR}/${download_path}"
```

Comment:
We may need to if we have to forcibly change the Maven-level version to include -odpi tag. Wouldn't know until we try to build it. But we would still be better off with a tracking repository under ODPi org on GitHub methinks.